From: Mark Wielaard <mjw@redhat.com>
Date: Wed, 18 Jun 2014 11:14:07 +0200
Subject: dwarves_fprintf: DW_TAG_mutable_type doesn't exist.

DW_TAG_mutable_type was a mistake in an early DWARFv3 draft and was
removed in the final version.

http://dwarfstd.org/ShowIssue.php?issue=050223.1

Signed-off-by: Mark Wielaard <mjw@redhat.com>
Signed-off-by: Arnaldo Carvalho de Melo <acme@redhat.com>

Origin: upstream, http://git.kernel.org/cgit/devel/pahole/pahole.git/commit/?id=943a0de0679a34b5e630f85cd01cca35c0d0e544
Bug-Ubuntu: https://launchpad.net/bugs/1378841
Bug-Debian: https://bugs.debian.org/764484
Last-Update: 2016-03-24

diff --git a/dwarves_fprintf.c b/dwarves_fprintf.c
index 4d8e0f4..b746864 100644
--- a/dwarves_fprintf.c
+++ b/dwarves_fprintf.c
@@ -75,7 +75,6 @@ static const char *dwarf_tag_names[] = {
 	[DW_TAG_unspecified_type]	  = "unspecified_type",
 	[DW_TAG_partial_unit]		  = "partial_unit",
 	[DW_TAG_imported_unit]		  = "imported_unit",
-	[DW_TAG_mutable_type]		  = "mutable_type",
 	[DW_TAG_condition]		  = "condition",
 	[DW_TAG_shared_type]		  = "shared_type",
 #ifdef STB_GNU_UNIQUE
-- 
cgit v0.10.1

